### DISCUSSION TITLE
How to create a region by clicking and dragging using Mouse

### DISCUSSION THREAD

**Discussion by fantasticoder** [katspaugh/wavesurfer.js#3608] [Q&A] [answered]
Hey guys,

I am gonna to create the region by clicking and dragging the mouse on waveform but I couldn't find the way to do that.
Is there anyone who can help me to achieve this feature?
Not sure if library already provides such features so it would be very appreciated if you can help me.

Thank you

**Comment by ManimMadhav** [comment, marked as answer]
check ```enableDragSelection()```  method. <br>
<a href="https://wavesurfer.xyz/docs/classes/plugins_regions.default.html#enableDragSelection">Refer this.</a>

**Comment by fantasticoder** [comment]
Thank you, it works.
